src/main/java/frc/robot/subsystems/shooter/FlywheelSubsystem.java

```diff
@@ -0,0 +1,94 @@
+package frc.robot.subsystems.shooter;
+
+import com.ctre.phoenix.motorcontrol.ControlMode;
+import com.ctre.phoenix.motorcontrol.FeedbackDevice;
+import com.ctre.phoenix.motorcontrol.NeutralMode;
+import com.ctre.phoenix.motorcontrol.SupplyCurrentLimitConfiguration;
+import com.ctre.phoenix.motorcontrol.VelocityMeasPeriod;
+import com.ctre.phoenix.motorcontrol.can.TalonFX;
+
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.RunCommand;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants.ConversionConstants;
+import frc.robot.Constants.RobotMap;
+/**
+   * Test FlywheelSubsystem to handle shooting and velocity controls
+   */
+public class FlywheelSubsystem extends SubsystemBase {
+  private TalonFX m_flywheelMotor;
+  private double velocity;
+
+
+  private final double kFlywheelKF = 0.0467;
+
+  public FlywheelSubsystem() {
+   m_flywheelMotor = new TalonFX(RobotMap.kFlywheelTalonFX);
+
+   m_flywheelMotor.configFactoryDefault();
+
+   m_flywheelMotor.configVoltageCompSaturation(12);
+   m_flywheelMotor.enableVoltageCompensation(true);
+   m_flywheelMotor.configNominalOutputForward(0);
+   m_flywheelMotor.configNominalOutputReverse(0);
+   m_flywheelMotor.configNeutralDeadband(0.01);
+   m_flywheelMotor.setNeutralMode(NeutralMode.Coast);
+
+   m_flywheelMotor.configSupplyCurrentLimit(new SupplyCurrentLimitConfiguration(true, 45, 80, 0.04));
+
+   m_flywheelMotor.configSelectedFeedbackSensor(FeedbackDevice.IntegratedSensor);
+   m_flywheelMotor.configVelocityMeasurementPeriod(VelocityMeasPeriod.Period_1Ms);
+   m_flywheelMotor.configVelocityMeasurementWindow(64);
+   configFeedbackGains();
+
+   setDefaultCommand(new RunCommand(this::stopFlywheel, this));
+
+   SmartDashboard.putNumber("Flywheel kP", 0);
+   SmartDashboard.putNumber("Flywheel kF", kFlywheelKF);
+   SmartDashboard.putNumber("Target RPM", 0);
+  }
+
+  @Override
+  public void periodic() {
+    SmartDashboard.putNumber("Flywheel RPM", getFlywheelVelocityRPM());
+    SmartDashboard.putNumber("Flywheel native units", m_flywheelMotor.getSelectedSensorVelocity());
+    SmartDashboard.putNumber("Flywheel current draw", m_flywheelMotor.getSupplyCurrent());
+  }
+
+  public void setPower(double power){
+    m_flywheelMotor.set(ControlMode.PercentOutput, power);
+  }
+
+  public double getFlywheelVelocityRPM(){
+    return fxToRPM(m_flywheelMotor.getSelectedSensorVelocity());
+  }
+
+  public void setFlywheelRPM(double rpm){
+    m_flywheelMotor.set(ControlMode.Velocity, rpmToFX(rpm));
+  }
+
+  public void stopFlywheel(){
+    setPower(0);
+  }  
+
+  public void configFeedbackGains(){
+    double kP = SmartDashboard.getNumber("Flywheel kP", 0);
```

Comment:
This should default to a constant in the Constants file which will be 0 for now.